docker/README.md

```diff
@@ -65,7 +67,9 @@ services:
     stop_signal: SIGINT
     environment:
       - SESSION_SECRET=mysupersecretkey
-    # - TZ=America/Los_Angeles  # uncomment and edit to match you timezone if you want logs time and dates to match your timezone instead of UTC.
+      # Uncomment if you want logs time and dates to match your timezone instead of UTC
+      # Available at https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+      #- TZ=America/New_York
     networks:
       - zwave
     devices:
```

Comment:
Should we mention here that it is recommended to use the `/dev/serial/by-id/...` path for USB sticks?
And having something like `/dev/zwave` **inside** the container would be nice for UX.

Reply:
> And having something like /dev/zwave inside the container would be nice for UX

What you mean?

Reply:
I've seen on the HA discord that some peeps call the stick `/dev/zwave` inside their container so it is always called the same way in the UI. Probably makes more sense when you also have a zigbee stick too you have to distinguish it from but I thought it was a neat idea.

Reply:
yep, makes sense, just need to change the name in the device bind so

Reply:
Yeah, I do this. Do we make the UI default to /dev/zwave to match? If so, should we just have it startup the first time with a recommended set of minimum settings so that it just works?

Reply:
Defaults are defined here: https://github.com/zwave-js/zwavejs2mqtt/blob/master/src/store/mutations.js#L8

Feel free to edit them, BTW those defaults are just for frontend, it means that at least the user has to press on save to confirm settings and start the application